### PR TITLE
Robust time parsing

### PR DESCRIPTION
**Pull request by ChristianBeilschmidt**
time parsing for values bc was not possible

### CODE REVIEW

**Comment by ChristianBeilschmidt** [comment]
[Time Slider](https://trello.com/c/wsPxp9Qk/418-time-slider)

File: README.md [hunk 1]
```diff
@@ -5,53 +5,71 @@
 
 This workspace contains the Geo Engine crates.
 
-# Development
+## Development
```
Comment (michaelmattig) [inline README.md:8]: Dependencies/lints etc. should be under "Development". So please indent them as well

**Review by michaelmattig — CHANGES_REQUESTED** [review]
readme fix could have been a separate PR

File: README.md (hunk 1 above)
Comment (ChristianBeilschmidt) [reply README.md:8]: d52a58c

**Pull request merged**